test/integration/current_category_test.rb

```diff
@@ -68,4 +68,10 @@ class CurrentCategoryTest < ActionDispatch::IntegrationTest
       assert_equal "Ruby", find(".menu__item[data-menu-item-state=current]").text
     end
   end
+
+  test "render description as Markdown" do
+    visit "/category/ruby"
+
+    assert_equal "https://www.ruby-lang.org/", find(".category-description a")[:href]
```

Comment:
I've fixed it via https://github.com/bm-sms/daimon-news/pull/485/commits/7bffc009466826b5b53113c0e763679814200865